docs/docs/docs.md

```diff
@@ -59,6 +59,21 @@ val user1 = Github[IO](httpClient, accessToken).users.get("rafaparadela")
 
 `user1` in this case is a `IO[GHResponse[User]]`.
 
+It is also possible to pass a custom GitHub configuration (e.g. a GitHub Enterprise server endpoint)
```

Comment:
I think this looks good but would need its own section title like `Using github4s with GitHub Enterprise` and need to be moved to the bottom.